website/docs/feature-flag-tutorials/dotnet/dotnet-examples.md

```diff
@@ -0,0 +1,397 @@
+---
+title: .NET Feature Flag Examples
+slug: /feature-flag-tutorials/dotnet/examples
+---
+
+In our [.NET feature flag tutorial](/feature-flag-tutorials/dotnet), we implemented a simple feature flag that could be turned on and off. This document will walk you through some common examples of using feature flags in .NET.
+
+We built many features into Unleash, our open-source feature flag platform, to address the complexities of releasing code. This tutorial will explore the following:
+
+- [Gradual Rollouts for .NET Apps](#gradual-rollouts-for-net-apps)
+- [Canary Deployments in .NET](#canary-deployments-in-net)
+  - [What is a canary deployment?](#what-is-a-canary-deployment)
+  - [How to do canary deployments with a feature flag in .NET?](#how-to-do-canary-deployments-with-a-feature-flag-in-net)
+  - [Configure strategy constraints for canary deployments](#configure-strategy-constraints-for-canary-deployments)
+- [Server-side A/B Testing in .NET](#server-side-ab-testing-in-net)
+- [Feature Flag Analytics and Reporting in .NET](#feature-flag-analytics-and-reporting-in-net)
+  - [Enable impression data events in .NET](#enable-impression-data-events-in-net)
```

Comment:
In the section linked to here, perhaps add an example of how to hook into the impression data events in the app using the SDK? Or is this article more about how to integrate with the admin API from .NET?

Reply:
Could also just be a link to the relevant section in the repo/docs

Reply:
@daveleek I'll let Alvin know that it'd be good to add an example of using impression data events in the app as a fast-follow to this PR. Based off the docs we have on it https://docs.getunleash.io/reference/sdks/dotnet#handling-events